client/src/components/Loading/index.js

```diff
@@ -14,7 +15,18 @@ const Loading = ({width, height}: Props) => (
     <LoadingComponent type="tail_spin" width={width} height={height} fill="#0062db" />
   </div>
 )
-
+export const EntityDetailLoading = () => (
+  <ListGroupItem className="list-row">
+    <div className="loadingsmall list-row-item">
+      <LoadingComponent type="tail_spin" width={15} height={15} fill="#0062db" />
+    </div>
+  </ListGroupItem>
+)
+export const ModalLoading = () => (
```

Comment:
I am not a big fan of exporting multiple components from one file. But maybe that's just a matter of preference :man_shrugging:

Reply:
I prefer it this way, since the components are very similar, but my preference is not very strong...

Reply:
Neither is mine, keep it as it is :smile: